packages/client/src/object/convertWireToOsdkObjects/createOsdkObject.ts

```diff
@@ -38,7 +39,7 @@ import type { PropertyDescriptorRecord } from "./PropertyDescriptorRecord.js";
 const objectPrototypeCache = createClientCache(
   function(client, objectDef: FetchedObjectTypeDefinition<any, any>) {
     return Object.create(
-      process.env.target !== "browser" ? OsdkCustomInspectPrototype : null,
+      process.env.target !== "browser" ? null : null,
```

Comment:
Can remove the ternary if we just decide to set the prototype to null

Reply:
Lets restore this one too